kyo-prelude/shared/src/test/scala/kyo/StreamTest.scala

```diff
@@ -63,6 +67,18 @@ class StreamTest extends Test:
                     Seq.fill(5)(1)
             )
         }
+
+        "exact amount" in pendingUntilFixed {
+            def emit(remaining: Int)(ack: Ack): Ack < Emit[Chunk[Int]] =
+                if remaining <= 0 then Reducible.eliminate[Abort[Nothing]](Abort.panic(new Exception("Boom!")))
```

Comment:
`Reducible.apply` should be accessible only within Kyo's package. Ideally, we should make it `private[kyo]`. Was the goal to panic without having a pending `Abort[Nothing]`? That's not possible since the effect needs to be tracked and handled, even if it's a panic. Another way to produce a panic is just using `throw` directly but I'd reserve that only for unexpected behaviors. Have you observed issues with the stream not pulling the correct amount as well?